cluster/saltbase/salt/kube-addons/kube-addon-manager.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   hostNetwork: true
   containers:
   - name: kube-addon-manager
-    image: gcr.io/google-containers/kube-addon-manager:v5
```

Comment:
The same version bump should also be done in cluster/images/hyperkube/static-pods/addon-manager.
Can you also add a note to this file about that, so it becomes easier to notice?